src/ui/marker.js

```diff
@@ -28,7 +28,13 @@ type Options = {
     pitchAlignment?: string
 };
 
+const defaultHeight = 41;
+const defaultWidth = 27;
+
 export const TERRAIN_OCCLUDED_OPACITY = 0.2;
+// Zoom levels to transition 'horizon' aligned markers in globe view.
+const ALIGN_TO_HORIZON_BELOW_ZOOM = 4;
+const ALIGN_TO_SCREEN_ABOVE_ZOOM = 6; // Can't be larger than GLOBE_ZOOM_THRESHOLD_MAX.
```

Comment:
Why not use the transition values from globe? Or at least making it a function of it (e.g. `ZOOM_TRANSITION_THRESHOLD - 1` or else...), so that changing the setting of the transition happens in the same place. Otherwise this is a bit fragile to change when updating the actual globe-mercator transition zoom threshold.

Reply:
> Why not use the transition values from globe? Or at least making it a function of it

That's not a bad idea!I think that if we increase the globe transition we don't necessarily want to change these values, they're relatively arbitrary. It's mostly coincidental that I found that IMO they look good matching the current globe transition.

I think I like your idea of an assertion!